org.cataclysmdda.CataclysmDDA.json

```diff
@@ -16,22 +16,10 @@
       "name": "cataclysm-tiles",
       "buildsystem": "simple",
       "build-options": {
-        "env": {
-          "MAKE_ARGS": "PREFIX=/app LANGUAGES=all USE_XDG_DIR=1 TILES=1 SOUND=1 RELEASE=1 RUNTESTS=0 ASTYLE=0 LINTJSON=0"
-        }
+        "env": { "MAKE_ARGS": "PREFIX=/app LANGUAGES=all USE_XDG_DIR=1 TILES=1 SOUND=1 RELEASE=1 RUNTESTS=0 ASTYLE=0 LINTJSON=0" }
```

Comment:
Also I don't believe you intended linting here either.

Reply:
Good catch, thanks!